core/src/com/unciv/ui/newgamescreen/PlayerPickerTable.kt

```diff
@@ -86,7 +94,13 @@ class PlayerPickerTable(val previousScreen: PreviousScreenInterface, var gamePar
         if (!locked) {
             playerTable.add("-".toLabel(Color.BLACK, 30).apply { this.setAlignment(Align.center) }
                     .surroundWithCircle(40f)
-                    .onClick { gameParameters.players.remove(player); update() }).pad(5f).right().row()
+                    .onClick {
+                        gameParameters.players.remove(player)
+                        if (previousScreen is GameParametersScreen) previousScreen.apply{
```

Comment:
We don't need to wrap this is an apply if it's just one line, just use 
previousScreen.mapEditorScreen.tileMap.stripPlayer(player)

Reply:
Done. Will be fixed in the next commit.